cmd/agent/daemon/daemon.go

```diff
@@ -77,6 +82,16 @@ func NewRunCommand(version string) *cobra.Command {
 		exportersQueueSize = pflag.Int("exporters-queue-size", 4096, "Exporters queue size")
 	)
 
+	goflag.Func("netflow-grouping", "Group netflow to reduce cardinality. Eg: src_addr|dst_addr to remove both source and destination ports", func(s string) error {
```

Comment:
Nit: I think the pflag way of doing this is using `Var` with a type that specifies the `Value` interface
https://pkg.go.dev/github.com/spf13/pflag#Var
https://pkg.go.dev/github.com/spf13/pflag#Value

Reply:
Ah, ok, I will try this one